lib/Github/HttpClient/Message/ResponseMediator.php

```diff
@@ -59,7 +59,7 @@ public static function getApiLimit(ResponseInterface $response)
         $remainingCalls = self::getHeader($response, 'X-RateLimit-Remaining');
 
         if (null !== $remainingCalls && 1 > $remainingCalls) {
-            throw new ApiLimitExceedException($remainingCalls);
+            throw new ApiLimitExceedException((int) $remainingCalls);
```

Comment:
Shouldn't this cast happen before we compare with `1`?

Reply:
Yes you are right, what should we do with the current return type of this method? It has `@return null|string` change it to `int` or return the original result of the getHeader call and keep the current return type?

Reply:
Shame PHP never added the `(?int)` casting I suggested. Would have been perfect for line 61. ;)

Reply:
Though, casting null to zero would be fine. We could then fix the return to change zero back to null. Give me a sec...

Reply:
https://github.com/acrobat/php-github-api/pull/1